.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - 'v*'
+  pull_request:
+    branches:
+      - master
+jobs:
+  Build:
```

Comment:
Move this job into a `release.yml`

https://github.com/search?q=org%3Ajazzband+path%3A**%2Frelease.yml+language%3AYAML&type=code